app/models/pglogical_subscription.rb

```diff
@@ -116,7 +109,7 @@ def backlog
   end
 
   def sync_tables
-    MiqPglogical.with_connection_error_handling { pglogical.sync_subscription(id) }
+    self.class.with_connection_error_handling { pglogical.sync_subscription(id) }
```

Comment:
With each of these `s/MiqPglogical/self.class/` changes, it is now calling via the method that is inherited from `MiqPglogical::ConnectionHandling` so that the proper call to `pglogical(false)` is made to the proper class.